### ISSUE TITLE
Automatic login

### ISSUE THREAD

**Issue by Wolfy76700** [valknight/Splatnet2-Rich-Presence#10]
Curently the script forces you to get a Splatnet cookie through mitmproxy, which is completely useless given that splatnet2statink can recover the Splatnet cookie from a simple in-browser login.

**Comment by valknight** [comment]
Hey, thanks for the heads up. I'll get to doing this at some point, but currently I'm working on other things, so this may take a few days for me to implement.

Alternatively, feel free to open a PR with this implemented.

**Comment by HHK3** [comment]
I can eventually help implementing the Simple In-Browser Login into the code, if that's OK. 😃

**Comment by valknight** [comment]
that would be rad thanks?

**Comment by HHK3** [comment]
Just opened up a RP! 😃

**Comment by valknight** [comment]
Fixed in https://github.com/valknight/Splatnet2-Rich-Presence/pull/11

**Comment by valknight** [comment]
Re-opened due to the login endpoint of another project being used without permission

**Comment by frozenpandaman** [comment]
See my comments https://github.com/valknight/Splatnet2-Rich-Presence/pull/11#issuecomment-517066256 & https://github.com/valknight/Splatnet2-Rich-Presence/pull/11#issuecomment-517069235 for more discussion on this. Just wanted to link them here to connect the threads. :)

tl;dr I'm all good with & happy to see my API being used (would be best to check with NexusMine too, though, for his – the whole cookiegen process requires both)! …as long as you 1) provide a unique user agent (i.e. the name of this project, not mine 😁), and 2) don't directly lift the login + cookiegen code from my script/repo (GPLv3) which doesn't permit usage with this repo's current license (MIT).
